lib/YaST/Kdump/FirmwareAssistedDumpPage.pm

```diff
@@ -0,0 +1,40 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Handles Kdump Firmware Assisted Dump Main Page.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+package YaST::Kdump::FirmwareAssistedDumpPage;
+use parent 'YaST::PageBase';
```

Comment:
parent will be `StartUpPage.pm`

Reply:
this new page will be created as well in `lib/Distribution/Opensuse/Tumbleweed.pm`